docs/create-packages/creating-a-package-dotnet-cli.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Create a NuGet package using the dotnet CLI
+description: A detailed guide to the process of designing and creating a NuGet package, including key decision points like files and versioning.
+author: karann-msft
+ms.author: karann
+ms.date: 07/09/2019
+ms.topic: conceptual
+---
+
+# Create a NuGet package using the dotnet CLI
+
+No matter what your package does or what code it contains, you use one of the CLI tools, either `nuget.exe` or `dotnet.exe`, to package that functionality into a component that can be shared with and used by any number of other developers. This article describes how to create a package using the dotnet CLI. To install the `dotnet` CLI, see [Install NuGet client tools](../install-nuget-client-tools.md). Starting in Visual Studio 2017, the dotnet CLI is included with .NET Core workloads.
```

Comment:
Disclaimer: I'll review the full PR later. 

> Honestly, I wouldn't be surprised if dotnet pack works on non-SDK projects using the pack targets, as I expect dotnet pack is probably nothing more than an alias to dotnet msbuild -t:pack

Sometimes it will, other times it won't depending on the targets imported by the old style csproj. 
dotnet msbuild uses a different reference for the resolution of imports than full framework msbuild does.